### PR TITLE
py-ffmpeg-python: new package

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.  Adding @adamjstewart since it is a python package.

File: var/spack/repos/builtin/packages/py-ffmpeg-python/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class PyFfmpegPython(PythonPackage):
+    """Python bindings for FFmpeg - with complex filtering support"""
+
+    homepage = "https://github.com/kkroening/ffmpeg-python"
+    pypi = "ffmpeg-python/ffmpeg-python-0.2.0.tar.gz"
+
+    version("0.2.0", sha256="65225db34627c578ef0e11c8b1eb528bb35e024752f6f10b78c011f6f64c4127")
+
+    depends_on("py-setuptools", type="run")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ffmpeg-python/package.py:17]: setuptools should be a build dep, missing build dep on pytest-runner, missing build/run dep on future

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by alecbcs** [comment]
@qwertos do you have time to address the review feedback or should we close this PR?

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**